recipes/metabat/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "metabat" %}
+{% set version = "0.32.4" %}
+{% set sha256linux = "6db36f8d43e4e7d246c4f5028979952d8f29ed5fe69ae8175fb4f2be98cb9a22" %}
+{% set sha256osx = "d17fcee3891f2f478379edff30c5eee98dd8d76b2d876859f12087828abb142a" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  # Linux binary
+  url: https://bitbucket.org/berkeleylab/{{ name }}/downloads/{{ name }}-static-binary-linux-x64_v{{ version }}.tar.gz  # [linux]
+  sha256linux: {{ sha256linux }}  # [linux]
+
+  # OSX binary
+  url: https://bitbucket.org/berkeleylab/{{ name }}/downloads/{{ name }}-static-binary-osx-elcapitan-x64_v{{ version }}.tar.gz  # [osx]
+  sha256: {{ sha256osx }}  # [osx]
```

Comment:
OK, I think I got it.
The compilation procedure is pretty ugly! Briefly, it downloads and compiles `samtools 1.2` to get `libbam.a` and `libhts.a` and some `*.h` at a specific location. Then the 3 executables (`contigOverlaps`, `jgi_summarize_bam_contig_depths` and `metabat`) are compiled are we're done.
Problems are :
- `boost 1.60.0` installs `icu 56` and we need `58` to compile the sources
- the SConsctruc file will be hard to patch, I'm thinking of rewriting a new Makefile (not hard if `libbam.a` and `libhts.a` are available)

HTSlib is already available via bioconda. `libbam.a` could be generate from Samtools, so I'll propose a PR for samtools which also build `libbam.a`